recipes/sphinxcontrib-programoutput/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "sphinxcontrib-programoutput" %}
+{% set version = "0.11" %}
+{% set sha256 = "cbec3ee1c3abd09e105115ab69cb5ade8ca1be9811565a844f973e93e0314837" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
```

Comment:
Is this a build requirement?

Reply:
I will try with Sphinx and try to remove the other ones.